src/eliminate_data_type.cpp

```diff
@@ -108,7 +108,8 @@ void eliminate_data_type::apply(module& m) const
                                                            "scatternd_add",
                                                            "scatternd_mul",
                                                            "scatternd_none",
-                                                           "select_module"};
+                                                           "select_module",
+                                                           "quantizelinear"};
```

Comment:
Why is this added here?

Reply:
Using eliminate_data_type with uint8 to handle inputs adds a convert to dynamicquantizelinear and breaks CI check in the onnxbackend tests

Reply:
I thought we only did this for `quant_dot`(which also should include `dot`, `convolution`, and `quant_convolution`). Why are we removing uint8 on qlinear?

Reply:
Hitting fails in CI with Onnxruntime backend with the quant_dot. I can't seem to repro locally but its showing up in jenkins

Reply:
@TedThemistokleous are you able to reproduce locally ? Or else maybe share error log from Jenkins ? 
IMO we shouldn't require removing uint8 for the quantizelinear if we already handle quant_dot or quant_conv.

Reply:
Didn't see it locally but I saw it on jenkins. Let me push the target fix first and ensure this does arise. I was happening with compile

Reply:
Removed this. Seems to be fine working locally.